WordPress/Classes/Utility/InteractiveNotificationsManager.swift

```diff
@@ -365,6 +377,8 @@ private extension InteractiveNotificationsManager {
                 return NSLocalizedString("View", comment: "Opens the post epilogue screen to allow sharing / viewing of a post.")
             case .shareEditPost:
                 return NSLocalizedString("Edit Post", comment: "Opens the editor to edit an existing post.")
+            case .twoFactorAuthApprove:
+                return NSLocalizedString("Approve", comment: "Approve login attempt (verb)")
```

Comment:
Should we clarify the comment a little more? "Approve two-factor login attempt (verb)"?